### PR TITLE
Add more Defend-centric combat feats

### LINKED ISSUE #132: Defend action needs more character options
I would like to have more affixes/feats/class feature that interact more with this Actions, otherwise it feels like the "lesser" version of both Block and Dodge

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Fixes #132.

### CODE REVIEW

File: rulebook/src/character-options/classes/alchemist/combat_feats.md [hunk 1]
```diff
@@ -65,3 +65,7 @@
 ## Extra Effort
 
 **Effects:** Whenever you use the Catalyze action, you may choose to spend an additional AP. If you do, you have advantage on any attacks made.
+
+## Blast Guard
+
+**Effects:** After you make an attack that targets yourself, but before the attack's effects resolve, gain the benefit of the Defend action twice.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/alchemist/combat_feats.md:71]:
```suggestion
**Effects:** After you make a damaging attack that targets yourself, but before the attack's effects resolve, gain the benefit of the Defend action twice.
```

File: rulebook/src/character-options/classes/paragon/combat_feats.md [hunk 2]
```diff
@@ -69,3 +69,13 @@
 ## Shield of Mercy
 
 **Effects:** Whenever you block an attack, remove one stack of each ailment from the ally with the lowest life (including yourself) within a 2 tile radius.
+
+## Bastion
+
+**Effects:** Whenever you use the Defend action, grant the absorption gained to all creatures within a 2 tile range, rather than just yourself.
```
Comment (plof27) [inline rulebook/src/character-options/classes/paragon/combat_feats.md:75]: So fucking cool. I love this

File: rulebook/src/character-options/classes/incarnation/combat_feats.md [hunk 3]
```diff
@@ -78,8 +79,13 @@
...
+
 **Effects:** While you are transformed, you gain access to the *Fervor* action. 1 emotion: Gain 1 AP.
 
 ## Masochism
 
 **Effects:** When you select this feat, choose an affliction. Whenever you gain emotion, gain 2 stacks of the chosen affliction
+
+## Simering Emotion
+
+**Effects:** When you use the Defend action, gain 1 emotion.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/incarnation/combat_feats.md:91]: This is broken with Fervor: infinite absorption!
```suggestion
**Effects:** The first time each turn that you use the Defend action, gain 1 emotion.
```

File: rulebook/src/character-options/classes/paragon/combat_feats.md [hunk 4]
```diff
@@ -69,3 +69,13 @@
...
+
+## Immovable Object
+
+**Prerequisites:** Stand your Ground class feature
+
+**Effects:** While affected by the *stand your ground* effect, after using the Defend action, trigger it a second time.
```
Comment (plof27) [inline rulebook/src/character-options/classes/paragon/combat_feats.md:81]: Also great

File: rulebook/src/character-options/feats/defensive-feats.md [hunk 5]
```diff
@@ -31,3 +31,19 @@
...
 
 **Effects:** Whenever you use the Dodge reaction, you may move into an empty tile within movement range 1.
+
+## Persistent
+
+**Effects:** Whenever you use the Defend action, also gain the benefit of the Recover action.
+
+## Prepared
+
+**Effects:** At the end of each of your turns, if you used the Defend action this turn, gain 1 AP.
```
Comment (plof27) [inline rulebook/src/character-options/feats/defensive-feats.md:41]: Subtle but good

File: rulebook/src/character-options/feats/defensive-feats.md [hunk 6]
```diff
@@ -31,3 +31,19 @@
...
+
+## Vigilant
```
Comment (plof27) [inline rulebook/src/character-options/feats/defensive-feats.md:43]:
```suggestion
## Blur
```
Not sure if this name is already taken, but I like it a lot here

**Pull request merged**